README.md

```diff
@@ -8,6 +8,7 @@ Built with [Chart.js](https://www.chartjs.org/)
 1. Run `npm run watch ngx-charts-on-fhir` to build the library and watch for changes.
 2. Run `npm start showcase` to start the local development server for the showcase app.
 2. Run `npm start cardiovascular-health` to start the local development server for the cardiovascular-health app.
+2. Run `npm start cardiovascular-patient-app` to start the local development server for the cardiovascular-patient-app.
```

Comment:
I don't think we need to list every app here. It might confuse people into thinking they need to start all of them at once. Let's just keep showcase here as an example and we can add instructions for each app to its own README.